src/pjpersist/datamanager.py

```diff
@@ -217,6 +228,7 @@ def check_for_conflict(e, sql):
         psycopg2.errorcodes.DEADLOCK_DETECTED
     )
     if e.pgcode in serialization_errors:
+        CONFLICT_TRACEBACK_INFO.traceback = repr(traceback.extract_stack())
```

Comment:
I'd prefer format_exception / print_exception because we have that replaced with zope.exceptions.exceptionformatter methods, which will add __traceback_info__